scalafmt-tests/src/test/resources/spaces/TrailingWhiteSpace.stat

```diff
@@ -0,0 +1,25 @@
+maxColumn = 80
+# Devos: be careful this file contains essential but invisible spaces.
+<<< Remove trailing spaces from variables
+object A {
+  val a = 1    
+  val b = 2    
+}
+>>>
+object A {
+  val a = 1
+  val b = 2
+}
+<<< Remove trailing tab-blank-tab combinations from variables
+object B {
+  val a = 1	 	
+  val b = 2	 	
+}
+>>>
+object B {
+  val a = 1
+  val b = 2
+}
+
```

Comment:
would you mind remove the 3 blank lines at the end?

Reply:
Done, thank you.  So you have something against trailing whitespace ;-) 
Guess I had expected scalafmt to remove those for me.  Must not be set in
the project .scalafmt.conf or I bungled it.